ui/raidboss/data/06-ew/raid/p5n.ts

```diff
@@ -102,6 +107,54 @@ const triggerSet: TriggerSet<Data> = {
       delaySeconds: 9,
       run: (data) => data.numStones = 0,
     },
+    {
+      id: 'P5N Topaz Cluster',
+      type: 'StartsUsing',
+      // 76E[7-A] are casts from the Topaz Stone placeholders
+      // 76E7 is 3.7s, 76EA is 9.7s
+      // Callout will call to start at 76EA -> move to 76E7
+      netRegex: NetRegexes.startsUsing({ id: '76E[7A]', source: 'Proto-Carbuncle' }),
+      durationSeconds: 7,
+      infoText: (data, matches, output) => {
+        // Coordinates range from [92.5, 107.5]
+        // Map to [-1, 1]
+        const x = Math.round((parseFloat(matches.x) - 100) / 7.5);
+        const y = Math.round((parseFloat(matches.y) - 100) / 7.5);
+
+        const directions: { [coord: string]: { [dir: string]: string } } = {
+          '0,1': Outputs.dirSW,
```

Comment:
Hmm, please don't hardcode string outputs here.  Can you:
* make this `'0,1': 'dirSW',`
* then down below:
```
const dir0Str = output[data.topazRayDirections[0]]!();
const dir1Str = output[data.topazRayDirections[1]]!();
return output.text!({ start: dir1Str, end: dir0Str });
```
* add dirSW/SE/NE/NW to `outputStrings`

In other words, add an indirection through output strings for the directions.